datadog/resource_datadog_logs_archive_order_test.go

```diff
@@ -0,0 +1,95 @@
+package datadog
+
+import (
+	"context"
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"testing"
+)
+
+func ArchiveOrderConfig() string {
+	return fmt.Sprintf(`
+resource "datadog_logs_archive_order" "archives" {
+	archive_ids = []
+}`)
+}
+
+func ArchiveOrderEmptyConfig() string {
+	return fmt.Sprintf(`
+resource "datadog_logs_archive_order" "archives" {
+}`)
+}
+
+func TestAccDatadogLogsArchiveOrder_basic(t *testing.T) {
+	rec := initRecorder(t)
+	accProviders, _, cleanup := testAccProviders(t, rec)
+	defer cleanup(t)
+	accProvider := testAccProvider(t, accProviders)
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Given that there is a unique resource behind it I don't think we want to use parallel tests.